### ISSUE TITLE
Timer

### ISSUE THREAD

**Issue by PrutzerNL** [rospogrigio/daikin_residential#73]
Hi,

Thanks for this great working solution! It works perfect!

I have i request:
Is it possible to make a timer so you can run the airco for 1 or 2 hours and after that it will power off automatic?

Thanks a lot!

**Comment by rospogrigio** [comment]
You can do it using Automations or Node-red I think...

**Comment by rgerhards** [comment]
Yup, you can. It's very easy. Crete a time helper in HA, and maybe a binary toggle. Then, when the toggle is hit, turn on AC and start timer. When timer expires, stop AC. The two actions you do with automations. Let me know if this is a problem for you, I'll see that I can create some YAML for it.

**Comment by PrutzerNL** [comment]
Hi RGerhards, If you could make a script, it would be very nice.
The wifi controllers or software from Daikin are not very stable. Sometimes the units are offline for a few minutes. (The wifi signal is very good). If possible it would be nice to write a check if the units are powered really off, and if not, it will try it again after a few minutes.
